sdks/java/io/hadoop/input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIOTest.java

```diff
@@ -472,7 +478,8 @@ public void testReadWithNullCreateRecordReader() throws Exception {
    */
   @Test
   public void testReadersStartWhenZeroRecords() throws Exception {
-    InputFormat<Text, Employee> mockInputFormat = Mockito.mock(EmployeeInputFormat.class);
+
```

Comment:
I don't see it fixed in the most recent version of the file - https://github.com/apache/beam/pull/2460/files#diff-aa7ab5dd37b6540d3c9f8b13ed4fd7f7L475